assignments/objects.js

```diff
@@ -18,27 +18,43 @@ const example = {
 }
 
 // Write your intern objects here:
+const interns = [{"id": 1, "name": "Mitzi", "email": "mmelloy0@psu.edu", "gender": "F"},
```

Comment:
Excellent work creating an array of objects! The challenge was designed for you to write out individual objects (which would have allowed easier access when adding those new functions to individual interns), but you handled it well!